.github/workflows/ci.yml

```diff
@@ -26,6 +26,5 @@ jobs:
         run: yarn test
 
   scan:
-    needs: lint_and_test
```

Comment:
is ok to remove this ?

Reply:
yes removing `needs: lint_and_test` to speed up the build per recommendation from Jake